monitoring/volume_collector.go

```diff
@@ -41,8 +41,8 @@ func NewVolumeCollector(
 
 	vc.sizeMetric = metricInfo{
 		Desc: prometheus.NewDesc(
-			prometheus.BuildFQName(longhornName, subsystemVolume, "usage_bytes"),
-			"Used storage space in bytes for this volume",
+			prometheus.BuildFQName(longhornName, subsystemVolume, "actual_size_bytes"),
```

Comment:
Separate this change into another commit.

Reply:
Thank you. Done